include/alpaka/core/AlignedAlloc.hpp

```diff
@@ -9,54 +9,22 @@
 
 #pragma once
 
-#include <alpaka/core/BoostPredef.hpp>
```

Comment:
Why are you removing this?

Reply:
Because it was no longer necessary, until I had to readd the special handling for Intel. Good catch, I will readd it!